src/store/reducer/MerketReducer.ts

```diff
@@ -0,0 +1,17 @@
+import { GET_MARKET } from '../actions/ActionTypes'
```

Comment:
There is a typo in file naming, should be MarketReducer instead of MerketReducer